.forge-snapshots/BinPoolManagerTest#testFuzzUpdateDynamicLPFee.snap

```diff
@@ -1 +1 @@
-32557
```

Comment:
will raise a PR to update this -- its changing because the input can be 0 

`_lpFee = uint24(bound(_lpFee, 0, LPFeeLibrary.TEN_PERCENT_FEE));`